src/util/index.ts

```diff
@@ -28,3 +28,47 @@ export function convertToSentenceCase(string: string): string {
             return c.toUpperCase();
         });
 }
+
+export const wrapText = (
```

Comment:
Why can't we let the browser do this? Can we not control the width of the title section and just let the browser flow the text in it?

Reply:
I googled and I see that plotly just doesn't support this yet.

Reply:
Haha........ I'm glad your Googling led to the same results I came across 😅 Or am I? I would love to not have to manually wrap

Reply:
I just tried again, just to be extra sure (trying to limit the container of the title), and I couldn't get it to work. I think it might have something to do with the entire plot being rendered as an svg???